src/views/customer/CustomerListView/index.tsx

```diff
@@ -23,7 +23,7 @@ const CustomerListView = () => {
       <Container maxWidth={false}>
         <Toolbar />
         <Box mt={3}>
-          <Results customers={customers} />
+          <CustomerIndex customers={customers} />
```

Comment:
It is too soon to rename to `<CustommerIndex`. Let's fine tune the naming conventions first. The word index is too technical and might conflict with package imports. More over someone would come up with somethning like list (not index).